demo/time-picker-basic-demos.html

```diff
@@ -75,6 +75,13 @@ <h3>Clear Button</h3>
       </template>
     </vaadin-demo-snippet>
 
+    <h3>Auto open disabled</h3>
+    <p>Dropdown is only opened when clicking the toggle button or pressing Up or Down arrow keys.</p>
```

Comment:
Clicking on the clock is not opening the dropdown.

Reply:
fixed